tests/__init__.py

```diff
@@ -375,11 +375,13 @@ def start_xdp(self):
 
         # This roughly resembles test-portals.c and glib's test behavior
         if os.getenv("XDP_UNINSTALLED", None):
-            builddir = Path(os.getenv("G_TEST_BUILDDIR") or "tests") / ".."
-            # meson has x-d-p in a different build directory than the autotools build...
-            xdp_path = builddir / "xdg-desktop-portal"
-            if not xdp_path.exists():
-                xdp_path = builddir / "src" / "xdg-desktop-portal"
+            xdp_path = os.getenv("XDG_DESKTOP_PORTAL_PATH", None)
+            if not xdp_path:
+                builddir = Path(os.getenv("G_TEST_BUILDDIR") or "tests") / ".."
+                # meson has x-d-p in a different build directory than the autotools build...
+                xdp_path = builddir / "xdg-desktop-portal"
+                if not xdp_path.exists():
+                    xdp_path = builddir / "src" / "xdg-desktop-portal"
```

Comment:
Any point in keeping the fall-back?

Reply:
the primary reason is that I don't want to clean this up until autotools is properly removed. Then we can look at what else relies on G_TEST_BUILDDIR and somesuch and remove all vestiges. Otherwise it's twice the work for little benefit.

Reply:
The Autotools build has been dropped.